apps/OboeTester/app/src/main/cpp/FormatConverterBox.cpp

```diff
@@ -28,6 +28,7 @@ FormatConverterBox::FormatConverterBox(int32_t numSamples,
     mSource.reset();
     switch (mInputFormat) {
         case oboe::AudioFormat::I16:
+        case oboe::AudioFormat::IEC61937:
```

Comment:
IEC61937 cannot go through the flowgraph because it will get corrupted. It must be "bit perfect".

Reply:
Yup! See QuickManager.cpp. This is for OboeTester so I think this is ok as is.